src/sentry/conf/server.py

```diff
@@ -2392,6 +2392,7 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     "openid",
     "profile",
     "email",
+    "org:superuser",
```

Comment:
Let's move this up next to the `org:*` scopes, and add a comment that this is only for superusers and it is added when the superuser session is valid.